chain/client/src/chunk_validation.rs

```diff
@@ -146,6 +147,62 @@ impl ChunkValidator {
         });
         Ok(())
     }
+
+    /// This function does two things
+    ///     1. Verifies if the chunk_endorsements have enough stake for chunk to be included in block.
+    ///     2. Collects the chunk_endorsement signatures from chunk_validators to include in block body.
+    /// We return an optional signature.
+    ///     None specifies we didn't have enough stake to include chunk in block.
+    ///     Some(signature) gives us the ordered chunk_endorsement signatures to include in block.
```

Comment:
Readability nit: this method doesn't run any verification, and doesn't collect signatures because... they are already collected :) Thus I think it is too much context for a reader.

Can we reword it like this?

> Called by block producer. Returns `Some(signatures)` if node has enough signed stake for the chunk represented by `ChunkHeader`. Signatures have the same order as `ordered_chunk_validators`, thus ready to be included in a block as is. Returns `None` otherwise.

Reply:
Updated comment